### PR TITLE
reformat, add content, fix typos

### PR DESCRIPTION
**Pull request by mattip**
break up lines
fix some typos and rework some of the sentences
add more ideas from https://github.com/numpy/numpy/wiki/Google-Season-of-Docs-2019

### CODE REVIEW

File: 2019/NumPy_ideas_list.md [hunk 1]
```diff
@@ -1,19 +1,64 @@
...
+not in very good shape. The User Guide and website (https://numpy.org) are not
+laid out well for guiding users new to NumPy. Nor do they make any attempt to
+address *different kinds* of users. Improving the quality of NumPy
+documentation will be very valuable to our millions of users!
+
+**How NumPy's documentation is built**: All our documentation and websites are
+built with [Sphinx](http://www.sphinx-doc.org). Sphinx generates static
+websites (making them easy to deploy) and provides extensive functionality to
+transform plain-text *reStructuredText* documents to html, as well as extract
+and cross-link documentation automatically from `docstrings in Python source
```
Comment (rgommers) [inline 2019/NumPy_ideas_list.md]: stray backtick here

File: 2019/NumPy_ideas_list.md [hunk 2]
```diff
@@ -22,23 +67,66 @@ Welcome, and thank you for taking an interest in NumPy! On this page we will fir
...
+- Producing a roadmap or list of work items for engaging the community in
+  further documentation work (we have a lot of contributor, channeling their
+  energy by setting the right goals and priorities could have a lot of impact).
+  This may include making explicit what different kinds of users need to see.
+- Rewriting a section of the User Guide (that can then serve as a template for
+  other/new sections).
+- Adding non-textual images or graphics to enhance the textual explanations
+- Updating out-of-date references and refactoring content to latest best
+  practices
+- Integrate the documentation more cleanly with the growing body of on-line
```
Comment (rgommers) [inline 2019/NumPy_ideas_list.md]: nitpick: Integrating instead of Integrate

File: 2019/NumPy_ideas_list.md [hunk 3]
```diff
@@ -22,23 +67,66 @@ Welcome, and thank you for taking an interest in NumPy! On this page we will fir
...
+  literature on scientific computing, data science, resources for learning
+  Python, NumPy, and performance considerations when writing code.
+
+We assume that the technical writer who will take on this project is not yet
+familiar with NumPy and its community. This means time will need to be built
+into the project plan at the start to get familiar with it. Mentors will be
+able to provide walkthroughs, set up time for discussing with or interviewing
+different kinds of end users and content experts. We are open to listening to
+writers' suggestions and contributions.*Some mentors have experience working
```
Comment (rgommers) [inline 2019/NumPy_ideas_list.md]: needs a space before "*Some"

File: 2019/NumPy_ideas_list.md [hunk 4]
```diff
@@ -22,23 +67,66 @@ Welcome, and thank you for taking an interest in NumPy! On this page we will fir
...
+with technical writers in a day-job-in-industry setting, but not in an open
+source setting - so we'll all learn from eachother!*
```
Comment (rgommers) [inline 2019/NumPy_ideas_list.md]: typo: eachother should be two words

**Review by rgommers — COMMENTED** [review]
Thanks Matti. Just a couple of very minor comments. New ideas look good.

File: 2019/NumPy_ideas_list.md (hunk 1 above)
Comment (mattip) [reply 2019/NumPy_ideas_list.md]: fixed

File: 2019/NumPy_ideas_list.md (hunk 2 above)
Comment (mattip) [reply 2019/NumPy_ideas_list.md]: ok

File: 2019/NumPy_ideas_list.md (hunk 3 above)
Comment (mattip) [reply 2019/NumPy_ideas_list.md]: fixed

File: 2019/NumPy_ideas_list.md (hunk 4 above)
Comment (mattip) [reply 2019/NumPy_ideas_list.md]: fixed

**Comment by rgommers** [comment]
Merged, thanks @mattip

**Pull request merged**